ModelVale/ModelVale/CustomComponents/HealthBarView.m

```diff
@@ -95,17 +95,12 @@ - (void) drawNoArcHealthBar: (CGFloat)endRoundness thickness: (CGFloat)thickness
     UIBezierPath* path = [UIBezierPath new];
     [path moveToPoint:self.leftTopPoint];
     [path addLineToPoint:self.rightTopPoint];
-//    [path addQuadCurveToPoint:rightMiddlePoint controlPoint:rightTopControlPoint];
-//    [path addQuadCurveToPoint:self.rightBottomPoint controlPoint:rightBottomControlPoint];
-    [path addArcWithCenter:rightMiddlePoint radius:self.barHeight*0.5 startAngle:M_PI_2 endAngle:3*M_PI_2 clockwise:YES];
+    [path addQuadCurveToPoint:rightMiddlePoint controlPoint:rightTopControlPoint];
+    [path addQuadCurveToPoint:self.rightBottomPoint controlPoint:rightBottomControlPoint];
     [path addLineToPoint:self.leftBottomPoint];
-//    [path addQuadCurveToPoint:leftMiddlePoint controlPoint:leftBottomControlPoint];
-//    [path addQuadCurveToPoint:self.leftTopPoint controlPoint:leftTopControlPoint];
-    [path addArcWithCenter:leftMiddlePoint radius:self.barHeight*0.5 startAngle:3*M_PI_2 endAngle:M_PI_2 clockwise:YES];
-//    [path addQuadCurveToPoint:endPoint controlPoint:CGPointMake(150, 0)];
-//    [path addCurveToPoint:endPoint controlPoint1:CGPointMake(150, 0) controlPoint2:CGPointMake(170, 0)];
-//    [path addQuadCurveToPoint:from controlPoint:pointsCenter];
-//    [path closePath];
+    [path addQuadCurveToPoint:leftMiddlePoint controlPoint:leftBottomControlPoint];
+    [path addQuadCurveToPoint:self.leftTopPoint controlPoint:leftTopControlPoint];
+    [path closePath];
     self.shapeLayer.path = path.CGPath;
 }
 
```

Comment:
I don't know how this file ended up on this branch. I originally wanted it only on the animateHealth branch until it works and I am ready to merge that branch to master. So this has some commented out code and such that I don't really want on master. Fixing this would take me a long time on my own, but I'd appreciate any help